content/docs/hooks-state.md

```diff
@@ -187,6 +187,14 @@ In a function, we already have `setCount` and `count` as variables so we don't n
   </button>
 ```
 
+Similiar to `this.setState()`, `setCount` also accepts an updater function:
+
+```js{1}
+  <button onClick={() => setCount((currentCount) => currentCount + 1)}>
```

Comment:
Could be simplified to `currentCount => currentCount + 1` (just dropped the parens).